packages/sdk/src/swap/chains.ts

```diff
@@ -10,17 +11,17 @@ type ChainFn = (
 
 const chainFactory =
   (chain: Chain): ChainFn =>
-  (network, env) =>
-    ({
-      chain,
-      name: Chains[chain],
-      isMainnet: !isTestnet(network),
-      requiredBlockConfirmations: isNotNullish(
-        env.ingressEgress.witnessSafetyMargins[Chains[chain]],
-      )
-        ? Number(env.ingressEgress.witnessSafetyMargins[Chains[chain]]) + 1
-        : undefined,
-    }) as ChainData;
+  (network, env) => ({
+    chain,
+    name: chain,
```

Comment:
`chain` and `name` hold the same value here. Could we merge them?

Reply:
not easily because removing one of the properties would be a breaking change in the interface of the sdk 🙈

Reply:
but i think its okay to have them, eg it would allow us to add arbitrum with `chain: 'Arbitrum'` and `name: 'Arbitrum One'` (which is the networks actual name i think)

Reply:
Ah okay, i thought these values are just used internally. Makes sense then

Reply:
> but i think its okay to have them, eg it would allow us to add arbitrum with `chain: 'Arbitrum'` and `name: 'Arbitrum One'` (which is the networks actual name i think)

Indeed!